pkg/sql/parser/sql.y

```diff
@@ -4318,6 +4324,24 @@ opt_table_with:
     return unimplemented(sqllex, "create table with oids")
   }
 
+opt_create_table_on_commit:
+  /* EMPTY */
+  {
+    $$.val = tree.CreateTableOnCommitUnset
+  }
+| ON COMMIT PRESERVE ROWS
+  {
+    $$.val = tree.CreateTableOnCommitPreserveRows
```

Comment:
probably `/* SKIP DOC */` here and below

Reply:
done!